mangaki/mangaki/tests/test_anilist.py

```diff
@@ -19,236 +20,184 @@ def read_fixture(filename):
-                status=200, content_type='application/json'
-            )
-
-        anime_list = self.anilist.get_user_list(AniListWorks.animes, 'mrsalixor')
-        animes = set(anime_list)
-        self.assertEqual(len(animes), 52)
-
-        manga_list = self.anilist.get_user_list(AniListWorks.mangas, 'mrsalixor')
-        mangas = set(manga_list)
-        self.assertEqual(len(mangas), 57)
-
-        for work_type in AniListWorks:
-            responses.add(
-                responses.GET,
-                urljoin(AniList.BASE_URL, 'user/aaaaaaaaaaaaa/{}list'.format(work_type.value)),
-                body='{"error":{"status":404,"messages":["No query results for model [App/AniList/v1/User/User] aaaaaaaaaaaaa"]}}',
-                status=404, content_type='application/json'
-            )
+    def test_get_mangalist(self):
+        responses.add(
+            responses.POST,
+            self.anilist.BASE_URL,
+            body=self.read_fixture('anilist/mrsalixor_anilist_mangalist.json'),
+            status=200,
+            content_type='application/json'
+        )
 
-        inexistant_user_animelist = list(self.anilist.get_user_list(AniListWorks.animes, 'aaaaaaaaaaaaa'))
-        inexistant_user_mangalist = list(self.anilist.get_user_list(AniListWorks.mangas, 'aaaaaaaaaaaaa'))
-        self.assertCountEqual(inexistant_user_animelist, [])
-        self.assertCountEqual(inexistant_user_mangalist, [])
+        anime_list = list(self.anilist.get_user_list(AniListWorkType.MANGA, 'mrsalixor'))
+        self.assertEqual(len(anime_list), 100)
 
     @responses.activate
     def test_insert_into_database(self):
-        self.add_fake_auth()
         artist = Artist(name='Ishihara Tatsuya').save()
 
         # Test insert AniListEntry into database
         responses.add(
-            responses.GET,
-            urljoin(AniList.BASE_URL, 'browse/anime'),
-            body=self.read_fixture('anilist/airing_summer_2017_trimmed.json'),
-            status=200, content_type='application/json'
-        )
-
-        seasonal = list(self.anilist.list_seasonal_animes(year=2017, season='summer'))
-        self.assertEqual(len(insert_works_into_database_from_anilist(seasonal)), 7)
-
-        # Test insert AniListRichEntry into database
-        responses.add(
-            responses.GET,
-            urljoin(AniList.BASE_URL, 'anime/20912/page'),
+            responses.POST,
+            self.anilist.BASE_URL,
             body=self.read_fixture('anilist/hibike_euphonium.json'),
-            status=200, content_type='application/json'
+            status=200,
+            content_type='application/json'
         )
 
-        hibike_entry = self.anilist.get_work_by_id(AniListWorks.animes, 20912)
-        hibike = insert_work_into_database_from_anilist(hibike_entry)
+        hibike_entry = self.anilist.get_work(search_id=20912)
+        # FIXME: properly mock the insertion of related works
```

Comment:
Do we really have to mock them? Can we not add them directly? And let them aside?